flocker/restapi/test/test_infrastructure.py

```diff
@@ -412,17 +418,20 @@ def test_decode(self, logger):
         render(app.app.resource(), request)
         self.assertEqual(objects, app.kwargs)
 
-    def assertNoDecodeLogged(self, logger, method):
+    def assertNoDecodeLogged(self, logger, method, path=b"/foo/bar"):
         """
         The I{JSON}-encoded request body is ignored when the given method is
```

Comment:
Hm.  How about a request body that's not JSON-encoded?  Is that important?  I suppose `null` and `12345` can both be loaded by the `json` module (but neither is actually technically legal).